extensions/amp-story/1.0/bookend/amp-story-bookend.js

```diff
@@ -206,41 +206,56 @@ export class AmpStoryBookend extends AMP.BaseElement {
     this.storeService_ = Services.storyStoreService(win);
   }
 
+  /** @private */
```

Comment:
`amp-story-bookend.build()` is only called from `amp-story`, somewhere in the layout process. These services are declared in `amp-story.constructor`, so technically it's impossible that the services are not defined yet at this point. What do you think? Have you seen it failing here?

Reply:
It's not the usage (in `build`) that's the problem, it's the instantiation (in the constructor).  I accidentally forgot to set them to null, but both `this.requestService_` and `this.storeService_` should be initialized to `null` in the constructor.